catalog/pytest.ini

```diff
@@ -34,6 +34,11 @@ filterwarnings=
     ignore:Calling `DAG.create_dagrun\(\)` without an explicit data interval is deprecated:airflow.exceptions.RemovedInAirflow3Warning
     ignore:.*is deprecated and will be (remoevd|removed) in Flask 2.3.:DeprecationWarning
     ignore:distutils Version classes are deprecated. Use packaging.version instead:DeprecationWarning
+# Reasons for filtering warnings
```

Comment:
This file is a little confusing because we still have the pre-existing comments explaining these warnings (starting on L21), and these new ones that duplicate the content. I think we could remove these and instead update the existing documentation with anything you'd like to add.